pinot-common/src/main/java/org/apache/pinot/common/metrics/ControllerGauge.java

```diff
@@ -26,6 +26,7 @@
  */
 public enum ControllerGauge implements AbstractMetrics.Gauge {
 
+  REPLICATION_FROM_CONFIG("replicas", false),
```

Comment:
Shall we also consider adding some of the static numbers as well?
E.g. the number of topic partitions.

Reply:
good idea. Will do that in a separate PR though, because to get partitions we need to create streamMetadataProvider. Will keeo the scope of this as static configs from the table config.